build/src/src/dyndnsClient/updateIp.js

```diff
@@ -0,0 +1,90 @@
+const httpGetRequest = require('./httpGetRequest');
+const EthCrypto = require('eth-crypto');
+const getKeys = require('./getKeys');
+const logs = require('../logs.js')(module);
+const db = require('../db');
+
+/**
+ * EthCrypto reference
+ *
+ * - Create private key
+ * const identity = EthCrypto.createIdentity();
+ * {
+      address: '0x3f243FdacE01Cfd9719f7359c94BA11361f32471',
+      privateKey: '0x107be946709e41b7895eea9f2dacf998a0a9124acbb786f0fd1a826101581a07',
```

Comment:
Avoid writing complete private public keys even in the comments

Reply:
Using '0x107be9...'